server/src/test/java/org/elasticsearch/cluster/routing/allocation/FailedShardsRoutingTests.java

```diff
@@ -633,8 +633,8 @@ public void testReplicaOnNewestVersionIsPromoted() {
 
         clusterState = ClusterState.builder(clusterState).nodes(
                 DiscoveryNodes.builder(clusterState.nodes())
-                .add(newNode("node3-6.x", VersionUtils.randomVersionBetween(random(), Version.V_6_0_0_alpha1, null)))
-                .add(newNode("node4-6.x", VersionUtils.randomVersionBetween(random(), Version.V_6_0_0_alpha1, null))))
+                .add(newNode("node3-7.x", VersionUtils.randomVersionBetween(random(), Version.V_7_0_0, null)))
```

Comment:
Can this be `Version.CURRENT.minimumIndexCompatibilityVersion()` instead of a hardcoded constant? I would then change the node name to something more generic. But I don't think we need hardcoded references to version constants here.